build.html

```diff
@@ -1,6 +1,6 @@
 {{#if dataSourceId}}
   {{#if formHtml}}
-    <form data-form-id="{{id}}" class="fl-form" data-form-uuid="{{uuid}}">
+    <form data-form-id="{{id}}" class="fl-form" data-form-uuid="{{uuid}}" novalidate>
```

Comment:
Without this, browser's validation takes over and creates problems because TinyMCE textareas are not "focusable". This then stops the validation code.